annotation-server/.env.example

```diff
@@ -1,24 +1,27 @@
-# Database Config
-ANNOTATION_DB_HOST=localhost
-ANNOTATION_DB_PORT=5432
-ANNOTATION_DB_USER=changeme
-ANNOTATION_DB_PASS=changeme
-ANNOTATION_DB_NAME=annotation_db
+########## Required for Development ##########
+
+# Google Sheets
```

Comment:
Is `EMPTY_GOOGLE_SHEET_ID` not also required if I want to be able to test my code locally?